src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsTextHelper.cs

```diff
@@ -9,24 +9,32 @@ namespace Swashbuckle.AspNetCore.SwaggerGen
     public static partial class XmlCommentsTextHelper
     {
         public static string Humanize(string text)
+        {
+            return Humanize(text, null);
+        }
+
+        public static string Humanize(string text, string xmlCommentEndOfLine)
         {
             if (text == null)
                 throw new ArgumentNullException(nameof(text));
 
+            // Using as a default behavior
+            xmlCommentEndOfLine ??= Environment.NewLine;
+
```

Comment:
This is a behavioural change - one is currently hardcoded to `"\r\n"` and needs to stay that way, not change to `Environment.NewLine`.

```suggestion
```